STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/registry_1_model.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+/********************************************************************
+ * This code was auto-generated from registry v1 DL and shall not be
+ * modified.
+ ********************************************************************/
+
+#ifndef REGISTRY_1_MODEL_H
+#define REGISTRY_1_MODEL_H
+
+#include "az_ulib_ipc_api.h"
+#include "az_ulib_result.h"
+#include "azure/az_core.h"
+
+#ifdef __cplusplus
+#include <cstdint>
+extern "C"
+{
+#else
+#include <stdint.h>
+#endif
+
+/*
+ * interface definition
+ */
+#define REGISTRY_1_INTERFACE_NAME "registry"
```

Comment:
>registry [](http://example.com/codeflow?start=35&length=8)

Consider an generic name for key values pairs, for example `map`.

Reply:
I think registry demonstrates the key value pair concept better than map, so I'm ok with keeping it. What do you think?